### PR TITLE
fix(button): added pf-m-aria-disabled state

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/2946

### CODE REVIEW

**Comment by jgiardino** [comment]
LGTM!

**Comment by mcoker** [comment]
@jgiardino thanks! Docs look good to me. Updated the PR.

**Comment by mcoker** [comment]
@christiemolloy updated

File: src/patternfly/components/Button/examples/Button.md [hunk 1]
```diff
@@ -61,41 +61,79 @@ import './Button.css'
...
-<br>
-<br>
+
+
 {{#> button button--modifier="pf-m-control" button--attribute="disabled"}}
   Control disabled
 {{/button}}
 ```
 
+```hbs title=aria-disabled
```
[the remark this answers is not in the document]
Comment (mcoker) [reply src/patternfly/components/Button/examples/Button.md]: Good call, updated.

**Pull request merged**